src/coreComponents/mesh/EdgeManager.cpp

```diff
@@ -33,33 +33,14 @@ using namespace dataRepository;
 
 EdgeManager::EdgeManager( string const & name,
                           Group * const parent ):
-  ObjectManagerBase( name, parent ),
-  m_edgesToFractureConnectorsEdges(),
-  m_fractureConnectorsEdgesToEdges(),
-  m_fractureConnectorEdgesToFaceElements()
+  ObjectManagerBase( name, parent )
 {
   registerWrapper( viewKeyStruct::nodeListString(), &m_toNodesRelation );
   registerWrapper( viewKeyStruct::faceListString(), &m_toFacesRelation ).
     setSizedFromParent( 0 );
 
   m_toNodesRelation.resize( 0, 2 );
 
-  registerWrapper( viewKeyStruct::edgesTofractureConnectorsEdgesString(), &m_edgesToFractureConnectorsEdges ).
-    setPlotLevel( PlotLevel::NOPLOT ).
-    setDescription( "A map of edge local indices to the fracture connector local indices." ).
-    setSizedFromParent( 0 );
-
-  registerWrapper( viewKeyStruct::fractureConnectorEdgesToEdgesString(), &m_fractureConnectorsEdgesToEdges ).
-    setPlotLevel( PlotLevel::NOPLOT ).
-    setDescription( "A map of fracture connector local indices to edge local indices." ).
-    setSizedFromParent( 0 );
-
-  registerWrapper( viewKeyStruct::fractureConnectorsEdgesToFaceElementsIndexString(),
-                   &m_fractureConnectorEdgesToFaceElements ).
-    setPlotLevel( PlotLevel::NOPLOT ).
-    setDescription( "A map of fracture connector local indices face element local indices" ).
-    setSizedFromParent( 0 );
-
```

Comment:
@CusiniM do you think I need to register them again on `FaceElementSubRegion`?

Reply:
hmm, I am not sure if they are ever accessed through the wrapper interface. I guess we can try not to. Although if not registered I don't think they ll be in restart files which is something we may want since it helps catching deviations in integratedTests.

Reply:
Yes, the current code compiles validates the integrated tests.
But I was thinking about the restart files indeed.
I bet I'll have to register and rebaseline then.

Reply:
yeah, I think you'll have to. Well, I d say you can go ahead and then this PR looks ready.